frontend/src/features/admin-form/responses/ResponsesPage/storage/useSecretKey.ts

```diff
@@ -17,7 +18,16 @@ type SecretKeyBroadcastMessage =
     }
 
 export const useSecretKey = (formId: string) => {
-  const [secretKey, setSecretKey] = useState<string>()
+  const { data: secretKey } = useQuery({
+    queryKey: ['secretKey', formId],
```

Comment:
should we define the queryKey in `frontend/src/features/admin-form/responses/queries.ts` under `adminFormResponsesKeys` instead (since the secret key is entered under that module)?

Reply:
@justynoh moved the query key to the the file, but do you prefer if whole query was extracted instead?

Reply:
I think keeping the query in this file is fine, we don't need to extract it. 

Actually, I was wondering if it's possible to avoid having an awkward `queryFn: () => Promise.resolve('')` at all. Instead of using `useQuery` with a dummy `queryFn`, can we just use `useQueryClient` with `QueryClient.getQueryData` and `QueryClient.setQueryData`?

Reply:
I havent tried it but I think in that case secretKey will not be reactive, and not respond to changes when setQueryData is called

Reply:
@justynoh updated with ìnitialData` option which i think you would prefer as its more declarative and no dummy query and supports type inference